pkg/plugin/dns/dns_linux.go

```diff
@@ -96,15 +94,13 @@ func (d *dns) eventHandler(event *types.Event) {
 	}
 	d.l.Debug("Event received", zap.Any("event", event))
 
-	responses := strings.Join(event.Addresses, ",")
-
-	// Update basic metrics. If the event is a request, the we don't need num_response, response, or return_code
+	// Update basic metrics
 	if event.Qr == types.DNSPktTypeQuery {
 		m = metrics.DNSRequestCounter
-		m.WithLabelValues(event.QType, event.DNSName).Inc()
+		m.WithLabelValues().Inc()
```

Comment:
You can move the `Inc()` call outside the if...else.

Reply:
Updated.